docs/openapi.yml

```diff
@@ -570,7 +570,7 @@ paths:
                 reference:
                   type: string
             example:
-              phone_number: "2028675309"
+              phone_number: "202-867-5309"
```

Comment:
Let's take this change out since it'll be fixed in [the other PR](https://github.com/GSA/notifications-api/pull/1721) and I left some feedback there! 🙂 

```suggestion
```